### PR TITLE
feat: tools/gitlab

### PR DESCRIPTION
**Pull request by wlrnet**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] Please open an issue before creating a PR or link to an existing issue
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

Fixes 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: api/core/tools/provider/builtin/gitlab/gitlab.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ identity:
     en_US: Gitlab
     zh_Hans: Gitlab
   description:
-    en_US: Gitlab plugin for commit
-    zh_Hans: 用于获取Gitlab commit的插件
+    en_US: Gitlab plugin,only support v4 api.
```
Comment (crazywoola) [inline api/core/tools/provider/builtin/gitlab/gitlab.yaml]:
```suggestion
    en_US: Gitlab plugin, API v4 only.
```

File: api/core/tools/provider/builtin/gitlab/gitlab.yaml [hunk 2]
```diff
@@ -5,8 +5,8 @@ identity:
...
+    zh_Hans: 用于获取Gitlab内容的插件,目前仅支持v4版本api。
```
Comment (crazywoola) [inline api/core/tools/provider/builtin/gitlab/gitlab.yaml]:
```suggestion
    zh_Hans: 用于获取 GitLab 内容的插件,目前仅支持 API v4。
```

File: api/core/tools/provider/builtin/gitlab/gitlab.yaml [hunk 3]
```diff
@@ -5,8 +5,8 @@ identity:
     en_US: Gitlab
```
Comment (crazywoola) [inline api/core/tools/provider/builtin/gitlab/gitlab.yaml]: Please replace all `Gitlab` with `GitLab`

File: api/core/tools/provider/builtin/gitlab/tools/gitlab_commits.yaml [hunk 4]
```diff
@@ -3,23 +3,23 @@ identity:
   author: Leo.Wang
   label:
     en_US: Gitlab Commits
-    zh_Hans: Gitlab代码提交内容
+    zh_Hans: Gitlab提交内容查询
```
Comment (crazywoola) [inline api/core/tools/provider/builtin/gitlab/tools/gitlab_commits.yaml]: There should be a space between Chinese words and English words.

**Review by crazywoola — CHANGES_REQUESTED** [review]
See comments.

File: api/core/tools/provider/builtin/gitlab/tools/gitlab_commits.yaml (hunk 4 above)
Comment (wlrnet) [reply api/core/tools/provider/builtin/gitlab/tools/gitlab_commits.yaml]: Okay I will make these change

File: api/core/tools/provider/builtin/gitlab/gitlab.yaml (hunk 3 above)
Comment (wlrnet) [reply api/core/tools/provider/builtin/gitlab/gitlab.yaml]: All of Gitlab has replaced to GitLab

**Review by crazywoola — APPROVED** [review]

**Pull request merged**